### DISCUSSION TITLE
Tab Notes

### DISCUSSION THREAD

**Discussion by ConstantinosM** [mbnuqw/sidebery#1240] [Ideas]
First of all THANK YOU for this excellent and customizable plugin!
I was wondering if there is a method to attach notes to a tab? I'm uncertain about the optimal implementation for this feature. One possibility could be displaying the tab title and notes when hovering over a tab. Alternatively, when viewing the list of tabs within a Sidebery folder/group, an additional field displaying the notes below each tab could be provided.

Ideas/suggestions?

**Comment by albino1** [comment]
You can follow this feature request:

https://github.com/mbnuqw/sidebery/issues/666

@emvaized made a fork with a proof of concept if you want to take a look or add feedback.

**Reply by ConstantinosM** [reply]
Thank you!
